src/thirdParty/register.js

```diff
@@ -48,20 +52,22 @@ export default function(type, name, item, overwrite = false) {
  */
 function isValidInput(type, name, item) {
   if (!type) {
-    console.warn('The type must be given in order to register.');
+    log('The type must be given in order to register.');
 
     return false;
   }
 
   if (!name) {
-    console.warn(`The ${type} must have a name in order to register.`);
+    log(`The %s must have a name in order to register.`, type);
```

Comment:
Is there a way to indicate that this is an error/warning state?

Reply:
Yep, done. I think there might be a conversation to be had / documentation to be written around how to use logging and when log, warn or error should be used.